packages/access-client/src/errors.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Indicates failure executing ability that requires access to a space that is not well-known enough to be handled.
+ * e.g. it's a space that's never been seen before,
+ * or it's a seen space that hasn't been fully registered such that the service can serve info about the space.
+ */
+export interface SpaceUnknown {
```

Comment:
we probably should extends `Failure` here same as other errors in ucanto

Reply:
I had omitted it on purpose so that access-client types didn't need to depend on ucanto (and we'd just rely on `tsc` checking the structural types), but don't have strong opinions on it either way, so I added the `extends` clause https://github.com/web3-storage/w3protocol/pull/391/commits/0b63c2e4bfd5890d7455042464bc6fd65459ffa7 .